src/style/global-style.js

```diff
@@ -34,6 +34,19 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+/* Remove outline for non-keyboard :focus */
+*:focus:not(:focus-visible) {
+  outline: none !important;
+  box-shadow: 0 0 0 rgb(255, 255, 255) !important;
+}
+
+*:focus-visible {
+  outline: 2px solid #286244 !important;
```

Comment:
In chrome, the download button has a `outline-offset: 1px` from the user agent style, for some strange reason. Resulting in:

![image](https://user-images.githubusercontent.com/7419664/164399081-6051a209-9e6f-40b8-95a0-b3ec980737fd.png)

So maybe just add `outline-offset: 0` here?